Blockzilla/BrowserViewController.swift

```diff
@@ -480,39 +481,55 @@ class BrowserViewController: UIViewController {
             UIKeyCommand(input: "]", modifierFlags: .command, action: #selector(BrowserViewController.goForward), discoverabilityTitle: UIConstants.strings.browserForward),
         ]
     }
+
+    func canShowTrackerStatsShareButton() -> Bool {
```

Comment:
It's probably smart to synchronize this, so it is only ever determined once. Or maybe have it determined once in a static block? Synchronization is costlier (though probably not practically) but easier to implement.

But all the logic from 490 to 515 should be in this method.